packages/financial-templates-lib/test/clients/OptimisticOracleClient.js

```diff
@@ -126,203 +127,418 @@ describe("OptimisticOracleClient.js", function () {
     );
   });
 
-  it("Basic proposal lifecycle: request, propose, expire without dispute", async function () {
```

Comment:
These tests are moved unmodified into a `describe()` block to denote that its working with the default `OptimisticOracle` contract.